components/sql/src/main/scala/pl/touk/nussknacker/sql/db/cassandra/CassandraMetaDataProvider.scala

```diff
@@ -1,17 +1,33 @@
 package pl.touk.nussknacker.sql.db.cassandra
 
 import pl.touk.nussknacker.sql.db.MetaDataProviderUtils
+import pl.touk.nussknacker.sql.db.query.UpdateResultStrategy
 import pl.touk.nussknacker.sql.db.schema._
 
 import java.sql.Connection
+import scala.util.Using
 
 class CassandraMetaDataProvider(getConnection: () => Connection) extends JdbcMetaDataProvider(getConnection) {
 
   private val queryHelper = new CassandraQueryHelper(getConnection)
 
-  override def getQueryMetaData(query: String): TableMetaData = throw new NotImplementedError(
-    "Generic query typing is not implemented for Cassandra"
-  )
+  override def getQueryMetaData(query: String, resultStrategyName: String): TableMetaData = {
+    val updateResultStrategyName = UpdateResultStrategy.name
+    if (resultStrategyName != updateResultStrategyName) {
```

Comment:
Jeszcze jest potrzebny test jakby to miało wejść

Reply:
English please ;P